### PR TITLE
Tembo CLI

### PR DESCRIPTION
**Pull request by sjmiller609**

### CODE REVIEW

File: landing/src/content/blog/2024-03-08-tembo-cli/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+slug: tembo-cli
+title: 'Announcing the Tembo CLI'
```
[the remark this answers is not in the document]
Comment (DarrenBaldwin07) [reply landing/src/content/blog/2024-03-08-tembo-cli/index.md:3]: Might be nice to use "entire" in here too - lots of tools to run postgres locally but not with extensions 

"Announcing Tembo CLI: Infrastructure as code for the entire Postgres ecosystem"

**Review by xenophenes — CHANGES_REQUESTED** [review]
Initial review of blogpost

File: landing/src/content/blog/2024-03-08-tembo-cli/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Using the Tembo CLI is easy. Just define your database(s) in a **tembo.toml** file.
+
+```toml
+[minimal]
+environment = "prod"
+instance_name = "minimal"
+```
+
+The above **tembo.toml** file includes one database. For additional information on how your **tembo.toml** file can be configured, please see "[Configure Tembo CLI](https://tembo.io/docs/tembo-cloud/Tembo-CLI/Getting_Started#configure-tembo-cli)" in the Tembo docs.
```
Comment (sjmiller609) [inline landing/src/content/blog/2024-03-08-tembo-cli/index.md:39]: maybe should link to examples instead, either here https://tembo.io/docs/tembo-cloud/Tembo-CLI/Examples/nextjs or here https://github.com/tembo-io/tembo/tree/main/tembo-cli/examples or combo

File: landing/src/content/blog/2024-03-08-tembo-cli/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+[vector-search]
+environment = "prod"
+instance_name = "vector-search"
+stack = "VectorDB"
+
+[analytics]
+environment = "prod"
+instance_name = "analytics"
+stack = "DataWarehouse"
+```
```
Comment (sjmiller609) [inline landing/src/content/blog/2024-03-08-tembo-cli/index.md:64]: maybe should indicate what it's like to connect to them locally
```
psql postgres://...@vector-search.local.tembo.io
psql postgres://...@analytics.local.tembo.io
```

**Comment by sjmiller609** [comment]
todo: replace illustration with new from design

File: landing/src/content/blog/2024-03-08-tembo-cli/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+slug: tembo-cli
+title: 'Announcing Tembo CLI: Infrastructure as code for the Postgres ecosystem'
+authors: [steven, adarsh, joshua]
+tags: [postgres, extensions, stacks]
+image: './dw_social.png'
+date: 2024-03-08T12:00
+description: Announcing Tembo CLI
+---
+
+# Announcing Tembo CLI: Infrastructure as code for the Postgres ecosystem
```
Comment (samay-sharma) [inline landing/src/content/blog/2024-03-08-tembo-cli/index.md:11]: Remove this. The title gets repeated twice.

**Review by samay-sharma — APPROVED** [review]
Left 2 minor comments, but after addressing them and updating the image and the date, @sjmiller609 feel free to merge whenever you think other stuff is ready to go.

**Pull request merged**